File: Web/TestUserinterface/Tests/BaseTest.cs
using Aquality.Selenium.Browsers;
using NUnit.Framework;

namespace TestUserinterface.Tests
{

    class BaseTest
    {
        public Browser browser;

        [SetUp]
        public void SetUpBrowser()
        {
            browser = AqualityServices.Browser;
            browser.Maximize();
        }

        [TearDown]
        public void TearBrowserDown()
        {
            browser.Quit();
        }
    }
}

File: Web/src/test/java/tests/BaseTest.java
package tests;

import com.userinyerface.utils.BrowserManager;
import org.testng.annotations.AfterTest;
import org.testng.annotations.BeforeTest;

public abstract class BaseTest {

    @BeforeTest
    public void setUp() {
        BrowserManager.getBrowser().maximize();
    }

    @AfterTest
    public void tearDown() {
        BrowserManager.getBrowser().quit();
    }
}
